Create ui-c8y-1022-4-10-user-with-sso-redirect-and-no-access-to-any-app-infinite.md (#3508)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-4-10-user-with-sso-redirect-and-no-access-to-any-app-infinite.md

---
date: ""

> title: User with SSO redirect and no access to any app infinite loop fix.

Reviewer: ```suggestion
title: Users with SSO redirect and without application access no longer run into an infinite loop
```

> title: User with SSO redirect and no access to any app infinite loop fix. [GRAFT][release/cd] (#9494)

Reviewer 2: ```suggestion
title: User with SSO redirect and no access to any app infinite loop fix.
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63961
version: 1022.4.10
---

> If user had SSO and SSO redirect enabled but had no permission to access any app, after login try he was falling into redirecting loop. Right now it is fixed and if user has no app to access, he is redirected to login view and error is shown.

Reviewer: ```suggestion
Previously, if users had SSO and SSO redirect enabled but did not have permission to access any application, they ran into an infinite loop after trying to log in. This issue has been fixed. If users have no application access, they are now redirected to the login page, and an error is shown.
```

> If user had SSO and SSO redirect enabled but had no permission to access any app, after login try he was falling into redirecting loop. Right now it is fixed and if user has no app to access, he is redirected to login view and error is shown.

Reviewer 2: ```suggestion
If user had SSO and SSO redirect enabled but had no permission to access any app, after login try he was falling into redirecting loop. Right now it is fixed and if user has no app to access, he is redirected to login view and error is shown.
```
